psk_fitness/psk_fitness/Interfaces/Services/IWorkoutService.cs

```diff
@@ -5,7 +5,7 @@ namespace psk_fitness.Interfaces.Services
 {
     public interface IWorkoutService
```

Comment:
Workout service uses userEmail and Workout repository uses userId. Why?

Reply:
Because the email is converted to userId in the service, just following what the IgnasJo's TopicService did

Reply:
No, TopicService did not touch on user's id. It refered only to email as id but no convertion was happening there.

Reply:
public async Task<List<TopicDTO>> GetUserTopicsAsync(string userEmail)
    {
        var user = await _userRepository.GetUserByIdAsync(userEmail);
        var userTopics = await _topicRepository.GetTopicsByUserIdAsync(user.Id);
        var userDisplayTopics = _mapper.Map<List<TopicDTO>>(userTopics);
        return userDisplayTopics;
    }
This is from TopicService, main branch, can you remind me who wrote this code again, we'll have to take care of him/her